src/JuliusSweetland.OptiKey/Enums/FunctionKeys.cs

```diff
@@ -67,6 +67,7 @@ public enum FunctionKeys
         LeftAlt,
         LeftCtrl,
         LeftShift,
+        RightShift,
```

Comment:
Could you also add equivalents in the KeyValues class for RightShift:

```
public static readonly KeyValue LeftShiftKey = new KeyValue(FunctionKeys.LeftShift);
```
and add to **KeysWhichCanBePressedDown** and **KeysWhichCanBeLockedDown**